uint32 seq
uint32 agent_num
uint32 agent_index
float64[] position_x
float64[] position_y
float64[] position_yaw
float64[] target_x
float64[] target_y